src/java.base/linux/native/libjava/CgroupMetrics.c

```diff
@@ -43,3 +44,12 @@ Java_jdk_internal_platform_CgroupMetrics_getTotalMemorySize0
     jlong page_size = sysconf(_SC_PAGESIZE);
```

Comment:
Preexisting: Check or at least assert for error? sysconf could also signal "indeterminable" state beside an error. I cannot see how either one could happen with _SC_PHYS_PAGES or _SC_PAGESIZE, so an assert would probably be enough.
(I know you do an assert in java, but that has to be switched on explicitly - do we run tests with asserts enabled?)

Reply:
> (I know you do an assert in java, but that has to be switched on explicitly - do we run tests with asserts enabled?)

Yes. That's why `-esa` is being added here:
https://github.com/openjdk/jdk/pull/12118/files#diff-597a72f91945acae784afe2610925a1da491f138eec44b98cb630414dac9cc76R204

I'll fix the other pre-existing issues you point out in another bug.